src/components/admin/app-navbar/components/dropdowns/LanguageDropdown.vue

```diff
@@ -22,20 +17,29 @@
 
 <script>
   import Vue from 'vue'
-  import { mapActions } from 'vuex'
 
   export default {
-    name: 'language-selector',
+    name: 'language-dropdown',
 
     props: {
       options: {
         type: Array,
-        required: true
+        default () {
+          return [
```

Comment:
this is incorrect.  there should be no real data in default props. Only placeholders and such.